Plugin/src/SofaPython3/PythonEnvironment.cpp

```diff
@@ -173,34 +173,34 @@ void PythonEnvironment::Init()
     if ( !Py_IsInitialized() )
     {
         msg_info("SofaPython3") << "Initializing python";
-        py::initialize_interpreter();
+        executePython([]{ py::initialize_interpreter(); });
```

Comment:
This is not python code (it is setting the python environment). I don't think it has to be in executePython.